src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -375,6 +381,26 @@ void ThrusterDynamicEffector::addThruster(THRSimConfig* newThruster, Message<SCS
 }
 
 
+/*! This method is used to update the blow down effects to the thrusters
+* at every computeForceTorque call when the thrusters are attached to a fuel
+* tank subject to blow down effects.
+ @return void
+ */
+void ThrusterDynamicEffector::ComputeBlowDownDecay()
+{
+    double currentFuelMass = this->fuelMass->getState()(0,0);
+    double massPercentage = currentFuelMass / this->maxFuelMass;
+
+    std::vector<THRSimConfig>::iterator it;
+    THROperation *ops;
+    for(it = this->thrusterData.begin(); it != this->thrusterData.end(); it++)
+    {
+        ops = &it->ThrustOps;
+        ops->BlowDownFactor = massPercentage;
```

Comment:
I think this makes sense for the initial linear model. Zero thrust at zero mass percentage. We aren't giving an option to have a generic y = mx + b model, but I don't think it is necessary. Maybe just explicitly say what type of model this is in a comment?

Reply:
Ultimately both this simplified version of the linear model and the new polynomial model given to us by structures could use the same code, so I'm implementing that.